src/Database/Redis/PubSub.hs

```diff
@@ -3,21 +3,39 @@
 
 module Database.Redis.PubSub (
     publish,
+
+    -- ** Subscribing to channels
+    -- $pubsubexpl
```

Comment:
No, I added an explanation of why there are two different pub/sub implementations.  That is a haddock reference to the documentation block at the bottom of the file.